configs/top_down/rsn/coco/single_ctf_rsn18_coco_256x192.py

```diff
@@ -59,9 +59,9 @@
     test_cfg=dict(
         flip_test=True,
         post_process=True,
-        shift_heatmap=True,
+        shift_heatmap=False,
```

Comment:
Will this change affect accuracies (if any) reported in model zoo?

Reply:
This is the config for RSN. And other models are not changed. So it will not affect the model zoo.